packages/datasources/javascript/test/test.js

```diff
@@ -0,0 +1,90 @@
+import { test } from 'uvu';
+import { getRunner } from '../index.js';
+import * as assert from 'uvu/assert';
+
+test('empty query should not run', async () => {
+	const runner = getRunner();
+	try {
+		await runner('', './test/files/empty.js');
```

Comment:
There's a tool called `mock-fs` that is better than having a bunch of files floating around:

Example Usage:
https://github.com/evidence-dev/evidence/blob/next/packages/lib/preprocess/src/partials/inject-partials.spec.js#L5-L9